.circleci/config.yml

```
@@ -106,7 +106,7 @@ jobs:
   run-tests:
     <<: *defaults_js
 
-    resource_class: xlarge
+    resource_class: 2xlarge
```

```suggestion
    resource_class: xlarge
```
This produce a side effect, as `2xlarge` for docker is not available for our project and it will uses as fallback a `medium` container, degrading performance. (https://app.circleci.com/pipelines/github/pagopa/io-app/10573/workflows/10db8ed4-402f-4b9e-93cc-7ef284484f6a/jobs/76358)

---

:money_with_wings: